src/zrender.ts

```diff
@@ -126,7 +127,16 @@ class ZRender {
         const handerProxy = (!env.node && !env.worker && !ssrMode)
             ? new HandlerProxy(painter.getViewportRoot(), painter.root)
             : null;
-        this.handler = new Handler(storage, painter, handerProxy, painter.root);
+
+        const useTargetSize = (opts.useCoarsePointer == null || opts.useCoarsePointer === 'auto')
+            ? isTouchDevice()
+            : !!opts.useCoarsePointer;
+        const defaultTargetSize = 44;
+        const targetSize = useTargetSize
```

Comment:
`targetSize` and `useTargetSize` should be renamed to `pointerSize` and `usePointerSize`?